android/app/src/main/java/com/microsoft/codepush/react/CodePushUtils.java

```diff
@@ -265,6 +271,108 @@ public static String tryGetString(ReadableMap map, String key) {
         }
     }
 
+    public static void unzipFile(File zipFile, String destination) throws IOException {
+        FileInputStream fis = null;
+        BufferedInputStream bis = null;
```

Comment:
I'd prefer if we didn't use acronym identifiers. How about "inputStream", "bufferStream" and "zipStream"? It obviously looks repetitive at the point of decleation but I think it makes the code easier to read when you're using the variables latter on. Also, do we need to initialize these to null or would the compiler do that for us?